src/main/kotlin/org/acejump/action/AceAction.kt

```diff
@@ -21,9 +22,21 @@ sealed class AceAction: DumbAwareAction() {
     action.presentation.isEnabled = action.getData(EDITOR) != null
   }
 
-  final override fun actionPerformed(e: AnActionEvent) =
-    e.getData(EDITOR)?.let { invoke(SessionManager.start(it)) } ?: Unit
-
+  final override fun actionPerformed(e: AnActionEvent) {
+    val editor = e.getData(EDITOR) ?: return
+    val project = e.project
+  
+    if (project != null) {
+      val openEditors = FileEditorManagerEx.getInstanceEx(project).splitters.selectedEditors
+        .mapNotNull { (it as? TextEditor)?.editor }
+        .sortedBy { if (it === editor) 0 else 1 }
+      invoke(SessionManager.start(editor, openEditors))
```

Comment:
Could this be simplified to `invoke(SessionManager.start(*openEditors.toTypedArray()))`?